### ISSUE TITLE
IsValueType resolves incorrectly for DeclaringType

### ISSUE THREAD

**Issue by yq8** [0xd4d/dnlib#103]
Hello,

I have a call instruction in my testapp which looks like this:
`IL_000d: call instance string [mscorlib]System.Byte::ToString(string)`

I like to determine if the _DeclaringType_ of this _MemberRef_ is a _ValueType_ or if it is not a _ValueType_.
When resolving it like this:
`bool isValueType = method.DeclaringType.IsValueType;`
it returns false, however since it is a _System.Byte_ it should definitly return true since a _Byte_ is a _ValueType_.

I also tried to resolve the _TypeDef_ first and try to check the _IsValueType_ property on it, but when resolving 
the _TypeDef_ it returns **null**.

So, long story short : How am I supposed to find out if this MemberRef's _DeclaringType_ is a _ValueType_ or not?

Regards
yq8dev

P.S.: Here is a testfile in case you like to check for yourself : https://www.sendspace.com/file/lfthlh

**Comment by 0xd4d** [comment]
Sounds like you haven't added a resolver. See the README.

**Comment by yq8** [comment]
Thanks.  That fixed it.
